components/root-actions/RootActivityLog.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import ActivityLog from '../admin-panel/sections/ActivityLog';
+import CollectivePickerAsync from '../CollectivePickerAsync';
+import { Box } from '../Grid';
+import StyledInputField from '../StyledInputField';
+
+const RootActivityLog = () => {
+  const [account, setAccount] = React.useState(null);
+  return (
+    <Box my={4}>
+      <StyledInputField htmlFor="activity-log-account" label="Account" flex="1 1">
+        {({ id }) => (
+          <CollectivePickerAsync inputId={id} onChange={({ value }) => setAccount(value)} collective={account} />
+        )}
+      </StyledInputField>
+      {account && <ActivityLog accountSlug={account.slug} />}
+    </Box>
+  );
+};
```

Comment:
@Betree : If we are adding this section to the root actions, shouldn't we make sure that we change the condition here, 

https://github.com/opencollective/opencollective-api/blob/00b4ad4b41d933d1c70a0b9bc9fe631e14942aa4/server/graphql/v2/query/collection/ActivitiesCollectionQuery.ts#L54

so that the root user can access the activities of any collective? Otherwise when searching for collectives we might end up with,

![image](https://user-images.githubusercontent.com/12435965/187000187-3afb5983-a7bc-4cf6-bfce-464984fb74c9.png)

Reply:
Yep, this was actually part of Friday's demo but I forgot to link & push the associate PR: https://github.com/opencollective/opencollective-api/pull/7894